wscript

```diff
@@ -6,7 +6,7 @@ import fnmatch
 import glob
 sys.path.insert(0, sys.path[0]+'/waf_tools')
 
-VERSION = '1.0.0'
+VERSION = '1.0'
```

Comment:
Please keep the patch version also. We might need it in the future and it does not hurt anywhere. Users can still query `v>=1.1` (i.e., ignoring the patch version).

Reply:
OK. I never use the patch version :) Do I do like MAJOR/MINOR/PATCH ?